docs/Getting Started Part 2/index.md

```diff
@@ -0,0 +1,339 @@
+| "method" | "MB.registerComponent" - the request is assigned to SDL Core's MessageBroker where the component name will be associated with the socket ID. Further, SDL Core will send messages related to the named component over the corresponding connection|
+|"componentName"| The name of the component being registered. Must correspond to the appropriate component name described in the current guidelines.|
+
+Example Request:
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 100,
+  "method": "MB.registerComponent",
+  "params": {
+    "componentName": "BasicCommunication"
+  }
+}
+```
+
+The possible componentNames are:  
+
+  * `BasicCommunication` - Generic interface responsible for a collection of information and user actions. Includes `UpdateAppList` and `OnAppRegistered`.
+  * `UI` - Interface responsible for RPC events and information made visible to the user. Includes `Show` and `Alert`. 
+  * `Buttons` - Interface responsible for RPC events and information related to hard and soft buttons in the vehicle. Includes `OnButtonPress` and `OnButtonEvent`.
+  * `VR` - Interface responsible for RPC events and information related to voice recognition. Contains RPC voice recognition counterparts to the UI component RPCs.
+  * `TTS` - Interface responsible for RPC events and information related to text to speech capabilities. Includes RPC `Speak`.
+  * `Navigation`- Interface responsible for RPC events and information related to navigation. Includes `StartStream` and `GetWayPoints`.
+  * `VehicleInfo`- Interface responsible for RPC events and information related to vehicle data. Includes `GetVehicleData` and `SubscribeVehicleData`.
+  * `RC` - Interface responsible for RPC events and information related to the [Remote Control Feature](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0065-remote-control.md).
+  * `AppService` - Interface responsible for RPC events and information related to the [App Services Feature](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0167-app-services.md).
+
+### Response
+
+SDL provides a JSON Response
+
+| Key | Value Info   |
+| :------------- | :------------- |
+| "id"      | The value from the corresponding request      |
+| "result" | Value of id multiplied by 10. HMI can treat this as a successful registration|
+
+Example Response:
+``` json
+{
+  "id": 100,
+  "jsonrpc": "2.0",
+  "result": 1000
+}
+```
+
+## Component Readiness Requests 
+
+Once the components are registered, the HMI must notify SDL Core that it is ready to begin further communication using the [BasicCommunication.OnReady](../basiccommunication/onready) notification.
+
+Upon receipt of the OnReady notification, SDL Core will begin checking the availability of the different HMI components via a chain of requests:
+
+  * `UI.IsReady` - The display availability
+  * `VR.IsReady` - The voice recognition module availability
+  * `TTS.IsReady` - The Text-To-Speech module availability
```

Comment:
In [line 77](https://github.com/smartdevicelink/sdl_core_guides/pull/59/files#diff-aad4ca0433fcaa1d53a08891f7587b7bR77), you use `text to speech`.  It could be different from `Text-To-Speech` because here you're referring to the actual module, but I just wanted to call out the difference and double check if these should be formatted in the same way.